### ISSUE TITLE
POS tagging accuracy (CoNLL 2017)

### ISSUE THREAD

**Issue by HITalexwang** [tensorflow/models#1236]
I have read the [paper](https://github.com/tensorflow/models/blob/master/syntaxnet/g3doc/conll2017/paper.pdf) for CoNLL 2017 and found the UAS and LAS reported [here](https://github.com/tensorflow/models/blob/master/syntaxnet/g3doc/conll2017/README.md). However, there are 8 treebanks without dev set in CoNLL 2017 (such as Slovenian-SST and Latin), whose results are also reported. So I wonder if you sampled the dev set of these treebanks from the training set or did you evaluate them by other method. 
Also, in the model described in the paper, POS is predicted with an LSTM tagger. However, I didn't find the POS tagging accuracy anywhere. Could you please report the POS tagging accuracy of the baseline [models](https://github.com/tensorflow/models/tree/master/syntaxnet/g3doc/conll2017) for CoNLL 2017?

**Comment by djweiss** [comment]
We'll update the paper with more precise info. We use the same training/tune/dev splits as the UD-Pipe CONLL2017 release -- for those 8 languages, that means we set aside 5% of the training set as tuning, and 10% of the training set as dev. We don't use the dev set to tune hyperparameters, so as to make dev set comparisons meaningful.

Also, we actually join all morphological tags together to form a "POS" tag that includes every possible tag for a given token. We then predict these joint tags. However, we do so only as an auxiliary regularization for the parser, similar to the Zhang & Weiss (2016) "stack-propagation" parser, with a ratio of 1:8 (pos-tagging to parsing) updates.

If you want to get predicted POS tags, we recommend training a separate single-task model without the "parser" component in the spec as the fastest way to get a very accurate POS tagger.
